src/api/services/todos.service.js

```diff
@@ -0,0 +1,33 @@
+const { add, format } = require("date-fns");
+const Todo = require("../../models/Todo");
+
+exports.addDateToTodo = async (id, date) => {
+  const todo = await Todo.findById(id).exec();
+
+  todo.addedInCalendar.set(date, {});
+  await todo.save();
+};
+
+exports.repeatTodo = async (id, date, type, week) => {
+  switch (type) {
+    case "everyDate":
```

Comment:
이 케이스가 헷갈립니다. 매일, 매월? 매주는 아래에 있어서 아닌 것 같은데 추가 설명 가능하실까요!

Reply:
everyDay, everyWeek 인 매일/매주를 뜻하신 것처럼 보이네요
 또 스위치문의 case는 상수 값이므로 대문자로 표기해주는 게 더 좋을 것 같아요!
 
 ```suggestion
    case "EVERY_DAY":
```

Reply:
수진님이 말씀하신대로 반복에 있어서, 매일 할지 매주같은요일에 할지에 대한 분기처리입니다.